crates/wasm/src/wasm_planner.rs

```diff
@@ -190,6 +79,7 @@ impl WasmPlanner {
     /// Set gas prices
     /// Arguments:
     ///     gas_prices: `GasPrices`
+    #[wasm_bindgen]
```

Comment:
for wasm_bindgen, it's not necessary adding annotations for struct methods. The rendered class will include them automatically.

Reply:
removed